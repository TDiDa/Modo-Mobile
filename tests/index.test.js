import pkg from '../package.json';

const testDist = process.env.LIB_DIR === 'dist';

describe('modo-mobile dist files', () => {
  it('exports modules correctly', () => {
    const modo = testDist ? require('../dist/modo-mobile') : require('../components'); // eslint-disable-line
    expect(Object.keys(modo)).toMatchSnapshot();
  });

  if (testDist) {
    it('should have modo.version', () => {
      const modo = require('../dist/modo-mobile'); // eslint-disable-line
      expect(modo.version).toBe(pkg.version);
    });
  }
});
