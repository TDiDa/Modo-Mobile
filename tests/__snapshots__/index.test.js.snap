// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modo dist files exports modules correctly 1`] = `
Array [
  "Button",
  "Icon",
  "NoticeBar",
  "Agree",
  "List",
  "ActivityIndicator",
  "version",
]
`;
